cli/main.go

```diff
@@ -1098,13 +1113,49 @@ func DefineSettings(seed *objects.Seed) ([]string, error) {
 	return settings, nil
 }
 
+//DefineResources defines any seed specified docker resource requirements.
+// Return []string of docker command arguments in form of:
+//	"-?? setting1=val1 -?? setting2=val2 etc"
+func DefineResources(seed *objects.Seed, size float64) ([]string, float64, error) {
+	var resources []string
+	var disk float64
+	
+	for _, s := range seed.Job.Interface.Resources.Scalar {
+		if s.Name == "mem" {
+			mem := s.Value + s.InputMultiplier * size
+			intMem := int64(math.Ceil(mem))
+			resources = append(resources, "-m")
+			resources = append(resources, fmt.Sprintf("%dm", intMem))
+		}
+		if s.Name == "disk" {
```

Comment:
Again I think a comment here or in func header (probably in header to accomdate both mem/disk) to indicate what the goal of the computation is would be justified. Basically, "Compute the total output disk / memory resource requirements as a function of input file size."